### PR TITLE
Long haul - elapsed time metric for operations

### PR DESCRIPTION
**Pull request by tmahmood-microsoft**

### CODE REVIEW

File: e2e/LongHaul/device/IotHub.cs [hunk 1]
```diff
@@ -169,10 +169,13 @@ public async Task ReportReadOnlyPropertiesAsync(Logger logger, CancellationToken
                     {
                         { "TotalTelemetryMessagesSent", _totalTelemetryMessagesSent },
                     };
+                    var sw = Stopwatch.StartNew();
```
Comment (drwill-ms) [inline e2e/LongHaul/device/IotHub.cs:172]: You can create the stopwatch above, and then use Reset and Stop in the loop.

That way the object isn't created over and over again.

File: e2e/LongHaul/device/LoggingConstants.cs [hunk 2]
```diff
@@ -18,6 +18,8 @@ internal static class LoggingConstants
         public const string TotalC2dMessagesCompleted = "TotalC2dMessagesCompleted";
         public const string TotalC2dMessagesRejected = "TotalC2dMessagesRejected";
         public const string C2dMessageDelaySeconds = "C2dMessageDelaySeconds";
+        public const string ReportedTwinUpdateDelaySeconds = "ReportedTwinUpdateDelaySeconds";
```
Comment (drwill-ms) [inline e2e/LongHaul/device/LoggingConstants.cs:21]: We talked about using the term `OperationSeconds` right?

File: e2e/LongHaul/service/DeviceOperations.cs [hunk 3]
```diff
@@ -59,10 +60,13 @@ public async Task InvokeDirectMethodAsync(Logger logger, CancellationToken ct)
                 {
                     try
                     {
+                        var sw = Stopwatch.StartNew();
```
Comment (patilsnr) [inline e2e/LongHaul/service/DeviceOperations.cs:63]: could the stopwatch also be created above here and reused each time we need it?

**Review by patilsnr — APPROVED** [review]

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**